.storybook/recipes/BaselineCard/BaselineCard.tsx

```diff
@@ -49,64 +58,133 @@ export interface Props {
+    }
+    const curr = cardRef.current;
+    curr.addEventListener('click', handleClick); /* 2 */
+
+    function handleClick() {
+      const isTextSelected = window.getSelection()?.toString(); /* 3 */
+      if (!isTextSelected) {
+        linkRef.current.click(); /* 4 */
+      }
+    }
+    /* 5 */
+    return () => {
+      curr.removeEventListener('click', handleClick, false);
+    };
+  }, []);
+
+  const componentClassName = clsx(
+    styles['baseline-card'],
+    linkHref && styles['baseline-card--clickable'],
+    className,
+  );
   return (
-    <Card className={componentClassName} {...other}>
-      <CardHeader className={styles['baseline-card__label']}>
-        {label}
-      </CardHeader>
-      <CardBody className={styles['baseline-card__body']}>{body}</CardBody>
-      {metadata && (
-        <CardFooter className={styles['baseline-card__footer']}>
-          <table className={styles['baseline-card__table']}>
-            <tbody className={styles['baseline-card__table-body']}>
-              <tr className={styles['baseline-card__table-row']}>
-                <th
-                  className={styles['baseline-card__table-label']}
-                  scope="row"
-                >
-                  Score
-                </th>
-                <td>
-                  <Tag
-                    className={styles['baseline-card__score']}
-                    hasOutline
-                    text={metadata.score}
-                    variant={metadata.variant}
-                  />
-                </td>
-              </tr>
-              <tr className={styles['baseline-card__table-row']}>
-                <th
-                  className={styles['baseline-card__table-label']}
-                  scope="row"
-                >
-                  Attempts
-                </th>
-                <td className={styles['baseline-card__table-data']}>
-                  {metadata.attempts}
-                </td>
-              </tr>
-              <tr className={styles['baseline-card__table-row']}>
-                <th
-                  className={styles['baseline-card__table-label']}
-                  scope="row"
-                >
-                  Line Passes On
-                </th>
-                <td className={styles['baseline-card__table-data']}>
-                  {metadata.deadline}
-                </td>
-              </tr>
-            </tbody>
-          </table>
-        </CardFooter>
-      )}
-    </Card>
+    /**
+     * 1) Wrapping the Card component in a container for attaching cardRef
+     * to the card without adding a ref prop to <Card />
+     */
+
+    <div className={styles['baseline-card__container']} ref={cardRef}>
+      <Card className={componentClassName} {...other}>
+        <CardHeader className={styles['baseline-card__label']}>
+          {label}
+        </CardHeader>
+        <CardBody className={styles['baseline-card__body']}>
+          {body}
+          {` `}
+          {linkHref && (
```

Comment:
Should this be
```suggestion
          {linkHref && linkText && (
```
? What happens currently if the user passes in `linkHref` without `linkText`?

Reply:
Good catch. Originally I had `linkText ? linkText : "Link"` but "Link" as link text is bad a11y... but I don't know what would make a good generic, catch-all default link text. Suggestions?

Reply:
I agree that's bad for accessibility and I think any default generic text would be bad – can we just make it required if there's a `linkHref`? And add that to the prop comments that both are required for the clickable version of the card to work.